generator/generator.yml

```diff
@@ -355,3 +355,40 @@ modules:
       - picoMobileMIB
       - picoIPv4MIB
       - picoIPv6MIB
+# Raritan MIB
+#
+# Used by Legran/Raritan PDUs
+#
+# http://support.raritan.com/px2/version-2.4.1/mibs/pdu2-mib-020400-39592.txt
+  raritan_pdu:
+    walk:
+      - 1.3.6.1.4.1.13742.6.5.2.4.1.4.1.1 # input voltage/current etc...  mono/tri phase(s)
+      - 1.3.6.1.4.1.13742.6.3.3.6.1.7.1.1 # input voltage/current decimal for each precedents entrys
+      - 1.3.6.1.4.1.13742.6.5.2.3.1.4.1   # input statistic
+      - 1.3.6.1.4.1.13742.6.3.3.4.1.7.1   # input statistic decimal
+      - 1.3.6.1.4.1.13742.6.5.4.3.1.4     # outlet voltage/current etc...
+      - 1.3.6.1.4.1.13742.6.3.5.4.1.7.1   # outlet decimal
+
+    version: 2
```

Comment:
These are defaults, no need to specify them.

Reply:
you talk about version of snmp ?

Reply:
Version and credentials.

Reply:
OK! I remove this options. No worry. Tomorrow

Reply:
Sorry for the time. I don't have access to my prometheus server and on my snmp pdu and devices. I can't change for the momment. I make change the 19 aout after my holiday.